### PR TITLE
Vote-736/style gov banner

### PR DESCRIPTION
**Pull request by SamiraMSadat**
## Jira ticket

[VOTE-736](https://cm-jira.usa.gov/browse/VOTE-736)

## Description

Style the official banner and language switcher for desktop and mobile.

### Post-deploy

1. lando retune
2. lando drush uli
3. Add a sitewide alert

### QA/Test

1. At desktop dimensions, verify the banner and language switcher match the designs. Be sure to expand the banner.
2. Switch to mobile, verify the banner and language switcher match the designs.
3. Verify the site-wide alert displays below the banner (both closed and expanded)

## Checklist for the Developer

- [x] A link to the JIRA ticket has been included above.
- [x] No merge conflicts exist with the target branch.
- [x] Automated tests have passed on this PR.
- [x] A reviewer has been designated.
- [x] Deployment and testing steps have been documented above, if applicable.

## Checklist for the Peer Reviewers

- [ ] The file changes are relevant to the task objective.
- [ ] Code is readable and includes appropriate commenting.
- [ ] Code standards and best practices are followed.
- [ ] QA/Test steps were successfully completed, if applicable.
- [ ] Applicable logs are free of errors.

### CODE REVIEW

**Comment by rayestrada** [comment]
Initial testing I observed
- The government banner trigger is overlapped by the language selector container
- the language dropdown isn't styled to match the current site

Screenshot 2024-02-05 at 9 53 17 AM

File: web/themes/custom/votegov/src/sass/uswds-overrides/_index.scss [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
 @forward "usa-breadcrumb";
 @forward "usa-form";
-@forward "usa-header";
 @forward "usa-legend";
 @forward "usa-label";
+@forward "usa-language-selector";
+@forward "usa-header";
 @forward "usa-button";
 @forward "usa-select";
+@forward "usa-language-banner";
```
Comment (SamiraMSadat) [inline web/themes/custom/votegov/src/sass/uswds-overrides/_index.scss:10]: Change the name of this file to usa-banner

File: web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-banner.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  }
+
+  @include at-media('tablet') {
+    @include u-height(8);
+  }
+}
+
+.banner__text-container {
+  @include u-height(6);
+  align-items: center;
```
Comment (SamiraMSadat) [inline web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-banner.scss:16]: Use the mixin

File: web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-selector.scss [hunk 3]
```diff
@@ -0,0 +1,71 @@
+@use "uswds-core" as *;
+
+.usa-language-container {
+  display: flex;
```
Comment (SamiraMSadat) [inline web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-selector.scss:4]: Use the mixins

File: web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-selector.scss [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  @include u-padding-x(2);
+  @include u-padding-y(1);
+
+  @include at-media('tablet') {
+    @include u-margin-top(3);
+    position: absolute;
```
Comment (SamiraMSadat) [inline web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-selector.scss:10]: Use the mixins

File: web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-selector.scss [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  }
+
+  @include at-media-max('tablet') {
+    @include u-height(8);
+    position: relative;
+    justify-content:center;
+    background-color: #fff;
+  }
+
+  .usa-button {
```
Comment (SamiraMSadat) [inline web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-selector.scss:24]: Remove button styles - they will be part of the global styles

File: web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-selector.scss [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+.usa-language__submenu {
+  @include u-padding(2.5);
+  @include u-width('auto');
+  @include u-radius('sm');
+  white-space: nowrap;
+
+  @include at-media-max('tablet') {
+    overflow: auto;
+    height: 80vh;
```
Comment (SamiraMSadat) [inline web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-selector.scss:49]: Double check this in the design

File: web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-selector.scss [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+.usa-language__primary-item:last-of-type .usa-language__submenu {
+  [dir="rtl"] & {
+    right: unset;
+    left: 0;
+  }
+}
+
+.language-switcher {
```
Comment (SamiraMSadat) [inline web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-selector.scss:69]: Check if we need this extra class

File: web/themes/custom/votegov/templates/block/block--votegov-languageswitcher.html.twig [hunk 8]
```diff
@@ -28,21 +28,23 @@
 
 {% if status_code != 'error' %}
   {% block content %}
-    <div class="usa-language-container usa-language--big">
+    <div class="usa-language-container usa-language--big grid-container">
       {% if content['#links']|length == 2 %}
         {{ content }}
       {% elseif content['#links']|length > 2 %}
+      <div class="language-switcher">
```
Comment (SamiraMSadat) [inline web/themes/custom/votegov/templates/block/block--votegov-languageswitcher.html.twig:35]: Can we use the div and class below instead?

**Review by rayestrada — APPROVED** [review]

**Pull request merged**